msgboard/templates/registration/logout.html

```diff
@@ -0,0 +1,8 @@
+<!--msgboard/templates/registration/logout.html-->
+  
+{% extends 'base_board.html' %}
+
+{% block content %}
```

Comment:
have you ever seen this kind of page on a real site? we should not need it.

Reply:
> have you ever seen this kind of page on a real site? we should not need it.

Had, and issue here, went to admin logout, w/o this page, any suggestions?